Luminary099/R30.agc

```diff
@@ -209,10 +210,10 @@ BOTHPAD		STCALL	RPADTEM
 			-TPER		# HPERMIN (300 OR 35) KFT.
```

Comment:
2 lines above, remove 1 space before `SAVE IT` (alignment with line change above)